moped-database/migrations/1653067649000_alter_table_moped_proj_milestones/up.sql

```diff
@@ -0,0 +1,18 @@
+-- add milestone_id column to moped_proj_milestones
+ALTER TABLE moped_proj_milestones ADD COLUMN milestone_id integer NOT null;
```

Comment:
if there were records in the `moped_proj_milestones` table, this looks like it would fail unless you set a default value or applied the `NOT NULL` constraint after you ran the update in L5?